src/Resources/ui/scss/structures/_cookie-bar.scss

```diff
@@ -4,18 +4,19 @@
 
 .kmcc-cookie-bar {
     position: fixed;
-    bottom: 0;
-    left: 0;
-    right: 0;
 
-    height: 100%;
+    bottom: 8px;
+    left: 8px;
+    width: calc(100vw - 16px);
```

Comment:
No need to calculate the width here, if you set the property "right": `right: 8px;`